astroid/nodes/node_classes.py

```diff
@@ -72,6 +72,19 @@ def _is_const(value):
     return isinstance(value, tuple(CONST_CLS))
 
 
+AssignedStmtsPossibleNode = Union["List", "Tuple", "AssignName", "AssignAttr", None]
+T = TypeVar("T")
+AssignedStmtsCall = Callable[
```

Comment:
```suggestion
T = TypeVar("T")

AssignedStmtsPossibleNode = Union["List", "Tuple", "AssignName", "AssignAttr", None]
AssignedStmtsCall = Callable[
```

We shouldn't directly mix TypeVars and Type Aliases. That just makes it difficult to read.

--
Another point, would it make sense to bound `T` to `nodes.NodeNG`? I.e.
```py
T_Nodes = TypeVar("T_Nodes", bound=NodeNG)
```
instead of `T`. That way we would limit the allowed types.

Reply:
I'll update!